bin/monit

```diff
@@ -5,4 +5,4 @@ root=$(python -c "import CMSMonitoring; print('/'.join(CMSMonitoring.__file__.sp
 MONIT_HOME="$(cd "$(dirname "$0")" && pwd)"
 export MONIT_DB_DICT_FILE="${MONIT_DB_DICT_FILE:-$MONIT_HOME/../static/datasources.json}"
 # run actual script
-python "$root/monit.py" ${1+"$@"}
+python "$root/monit.py" ${1+"$@"}
```

Comment:
there is no need to modify the file, leave it intact.

Reply:
Valentin,
The monit binary looks same. I am not sure why any change appeared.

Reply:
it is your editor which may add newline, etc. So any file you can revert to previous form by using `git checkout -- <file_name>`. But this is minor detail, I can merge the PR once you'll fix the log

Reply:
Valentin,
I have updated both files. Please review. Thanks for the git help, i will keep in mind next time.